README.md

```diff
@@ -9,21 +9,36 @@
 ⚠️ [Still using Swift 2.x?][]
 
 ## What is ReactiveSwift?
-__ReactiveSwift__ offers composable, declarative and flexible primitives that are built around the grand concept of ___streams of values over time___. These primitives can be used to uniformly represent common Cocoa and generic programming patterns that are fundamentally an act of observation, e.g.:
+__ReactiveSwift__ offers composable, declarative and flexible primitives that are built around the grand concept of ___streams of values over time___.
 
- * Delegate methods
- * Callback blocks
- * Notifications
- * Control actions and responder chain events
- * [Futures and promises](https://en.wikipedia.org/wiki/Futures_and_promises)
- * [Key-value observing](https://developer.apple.com/library/mac/documentation/Cocoa/Conceptual/KeyValueObserving/KeyValueObserving.html) (KVO)
+These primitives can be used to uniformly represent common Cocoa and generic programming patterns that are fundamentally an act of observation, e.g. delegate pattern, callback closures, notifications, control actions, responder chain events, [futures, promises](https://en.wikipedia.org/wiki/Futures_and_promises) and [key-value observing](https://developer.apple.com/library/mac/documentation/Cocoa/Conceptual/KeyValueObserving/KeyValueObserving.html) (KVO).
 
 Because all of these different mechanisms can be represented in the _same_ way,
-it’s easy to declaratively chain and combine them together, with less spaghetti
+it’s easy to declaratively compose them together, with less spaghetti
 code and state to bridge the gap.
 
-For more information about the concepts in ReactiveSwift, see the [Framework
-Overview][].
+### Core Reactive Primitives
+| Primitive | Description |
+| --------- | ------------|
+| `Signal` | `Signal` is a unilateral stream of events.<br /><br />It is like a live TV feed - you can observe and react to the content, but you cannot have a side effect on the live feed. |
```

Comment:
I don't think _unilateral_ is the right word here. _Unidirectional_, maybe?

The ` - ` should should be replaced by `—` here and below.